dist/index.d.ts

```diff
@@ -4286,7 +4286,11 @@ interface Room {
     /**
      * Returns an array of events happened on the previous tick in this room.
      */
-    getEventLog(raw?: boolean): EventItem[];
+    getEventLog(raw?: false): EventItem[];
+    /**
+     * Returns an raw JSON string of events happened on the previous tick in this room.
+     */
+    getEventLog(raw: true): string;
```

Comment:
It's better to put the more general signatures after more specific signatures

see https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#ordering

Here is also an another reason to do so:

Currently, when handling overloaded functions, typescript's built-in tool type `Parameter` would only return the last overload's params.